modules/crud/sql/read.go

```diff
@@ -69,18 +69,30 @@ func (s *SQL) Read(ctx context.Context, project, col string, req *model.ReadRequ
 				orderBys = append(orderBys, exp)
 			}
 			query = query.Order(orderBys...)
-
 		}
 	}
+	
+	switch req.Operation {
+	case utils.Count:
+		query = query.Select(goqu.COUNT("*"))
+	case utils.Distinct:
+		distinct := req.Options.Distinct
+		if distinct == nil {
+			return nil, utils.ErrInvalidParams
+		}
+		query = query.SelectDistinct(*distinct)
+	case utils.One, utils.All:
+		query = query.Select(selArray...)
```

Comment:
What will happen if select array is empty? Do we get the results?

Reply:
Yes